backend/FwLite/FwDataMiniLcmBridge.Tests/MiniLcmTests/ComplexFormComponentTests.cs

```diff
@@ -39,4 +39,84 @@ await fwDataApi.Cache.DoUsingNewOrCurrentUOW("Add ComplexFormEntryRef",
                 return ValueTask.CompletedTask;
             });
     }
+
+    [Fact]
+    public async Task DuplicateComplexFormComponents_AreNotDuplicated()
+    {
+        await AddDuplicateComponent();
+
+        var entry = await Api.GetEntry(_complexFormEntryId);
+        entry.Should().NotBeNull();
+        entry.Components.Should().HaveCount(1);
+        var component = await Api.GetEntry(_componentEntryId);
+        component.Should().NotBeNull();
+        component.ComplexForms.Should().HaveCount(1);
```

Comment:
Interesting. It seems Lcm is already handling this for us under the scenes eh?

Reply:
Yeah because this just calculated on the fly they can do that here.